DependencyInjection/Configuration.php

```diff
@@ -27,7 +27,9 @@ public function getConfigTreeBuilder()
         $rootNode->children()
```

Comment:
please add the information that you added in the README.md below as "info" to all config nodes here.
e.g. `->scalarNode('redirect_route')->defaultValue('fos_user_profile_show')->info("route to redirect after email confirmation")->end()`